x-pack/plugin/src/test/resources/rest-api-spec/test/runtime_fields/10_keyword.yml

```diff
@@ -14,7 +14,7 @@ setup:
               temperature:
                 type: long
               voltage:
-                type: float
+                type: double
```

Comment:
why this?

Reply:
Sneaky rounding error in the multiplication in the test script had me waste an hour trying to find out why 5.1 * 10 == 52. Stupid floating point.